index.html

```diff
@@ -52,13 +52,31 @@ <h1 class="header-title">
         <div class="application-demo__container">
           <img
             class="application-demo__img"
-            src="/images/Group 53 (1).png"
+            src="images/Group 53 (1).png"
             alt="application demo"
           />
           <hr class="application-demo__line" />
         </div>
       </div>
     </section>
-    <section>lcnj;sbc;aiygc;iy</section>
+    <section class="description">
+      <div class="container">
+        <h1 class="description-text">How the app works</h1>
+      </div>
+    </section>
+    <section class="description-login">
+      <div class="container">
+        <div class="header-description">
+          <img class="description-login__img" src="images/Rectangle (1).png" alt="description-login__img">
+            <div class="description-login__start">
+              <h3 class="description-text__login description-text__account">Create an account</h3>
+              <h1 class="description-text__login description-text__faststart">Create/login to an existing<br>
```

Comment:
title here is okay but it's not first level title, let's use second.
```suggestion
              <h2 class="description-section__title">Create/login to an existing<br>
```